internal/namespace/typesystem.go

```diff
@@ -474,6 +475,97 @@ func (nts *TypeSystem) typeSystemForNamespace(ctx context.Context, namespaceName
 	return BuildNamespaceTypeSystem(nsDef, nts.lookupNamespace)
 }
 
+// Returns the relation names (right-hand side) of an arrow expression given its left hand-side value
+//
+// Sample:
+//
+// ```
+// definition resource {
+//     relation reader: user
+//     relation writer: user
+//     relation parent: resource_group | resource
+
+//	    permission read = reader + writer + parent->read
+//	    permission write = reader
+//	}
+//
+// ```
+// Calling ResolveArrowRelations('parent') returns ['read']
```

Comment:
Can you extend the example to have multiple returned?

Reply:
done